salt/utils/systemd.py

```diff
@@ -18,6 +18,17 @@
 log = logging.getLogger(__name__)
 
 
+def _has_systemd_support():
+    try:
+        output = subprocess.check_output(("systemctl",), shell=False).decode().lower()
+        return "systemd" in output
+    except FileNotFoundError:
+        return False
+
+
+HAS_SYSTEMD = _has_systemd_support()
```

Comment:
`salt.modules.systemd` uses another approach to check systemd presence on the system.
```
salt.utils.systemd.booted(__context__)
```
Is this an expected difference?

Reply:
yea your right